### DISCUSSION TITLE
Coding Questions

### DISCUSSION THREAD

**Discussion by bpepple** [krateng/maloja#84] [Ideas]
In my free time I was thinking about adding some test coverage to the project and had a few questions before proceeding:

1. Does the project use any kind of formatting (ex. black, etc)?
2. Any interest in using pre-commit & flake8 for style guide enforcement.
3. Is there any interest in using one of the python package managers (poetry, pipenv, conda, etc) to make development easier?

Thx.

**Comment by krateng** [comment]
I'm not using any style guide enforcement or formatting so far. Since it's not a huge project and mostly done by one person, I'm not necessarily planning for anything too complex in that direction, but I could take a look at black which seems simple enough.

As for the package managers, is the issue the quick creation of a development environment with the correct dependencies?
